src/flexbuffer_json-inl.h

```diff
@@ -940,6 +940,19 @@ inline std::vector<std::string> JsonObject::get_as_string_array( const std::stri
     }
     return ret;
 }
+inline std::set<std::string> JsonObject::get_as_string_set( const std::string &name ) const
+{
+    std::set<std::string> ret;
+    if( has_array( name ) ) {
+        JsonArray ja = get_array( name );
+        for( JsonValue jv : get_array( name ) ) {
+            ret.insert( jv );
+        }
+    } else if( has_string( name ) ) {
```

Comment:
I don't understand why the desired behavior one function up from here in `get_as_string_array` for a missing key is to return `[""]` instead of `[]`. If there's a reason for that, maybe this should return a set containing an empty string, instead of an empty set?

Reply:
Because that's what the original json implementation did, and when I tried to do the 'obviously correct' thing, something broke.

```
std::vector<std::string> TextJsonObject::get_as_string_array( const std::string_view name ) const
{
    std::vector<std::string> ret;
    if( has_array( name ) ) {
        for( const std::string entry : get_array( name ) ) {
            ret.push_back( entry );
        }
    } else {
        ret.push_back( get_string( name ) );
    }
    return ret;
}
```